src/Compilers/CSharp/Portable/Binder/Binder_WithExpression.cs

```diff
@@ -0,0 +1,183 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    /// <summary>
+    /// This portion of the binder converts an <see cref="ExpressionSyntax"/> into a <see cref="BoundExpression"/>.
```

Comment:
Seems like it converts a `WithExpressionSyntax` specifically.

Reply:
I don't think doc comments on parts of a partial "work" properly.  This could be an ordinary non-doc comment.

---
In reply to: [413343653](https://github.com/dotnet/roslyn/pull/43249#discussion_r413343653) [](ancestors = 413343653)